tests/treeTest.py

```diff
@@ -25,5 +22,6 @@ def test_extract(self):
 
     def test_graph(self):
         tr = Tree(self.FL)
-        testChild = tr.root.child['tests'].child.get('specs', None)
-        self.assertTrue(testChild)
+        # use `set` to ignore the  order in comparison.
+        testChild = set(tr.root.child.keys())
+        self.assertEqual(testChild, set(os.listdir('tests')))
```

Comment:
Here you can use [assertCountEqual](https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual) instead of converting to sets.